amethyst_assets/src/progress.rs

```diff
@@ -144,6 +144,17 @@ impl Tracker for ProgressCounterTracker {
         asset_name: String,
         error: Error,
     ) {
+        error!(
+            "Error loading handle {}, {}, with name {}, caused by: {:?}",
+            handle_id,
+            asset_type_name,
+            asset_name,
+            error
+        );
+        error
+            .iter()
+            .skip(1)
+            .for_each(|e| error!("caused by: {:?}", e));
```

Comment:
Probably would want to concatenate these two `error!(...)` into one just as a caution against possible out of order printing.